src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
```

Comment:
u can use this file `react-app-env.d.ts` for all types and there won't be need in importing